shopelectro/tests/tests_js_analytics.py

```diff
@@ -29,8 +36,182 @@ def test_google_ecommerce_purchase(self):
         success_page.wait_loaded()
         self.assertTrue(success_page.is_success())
 
-        order = Order.objects.order_by('-created').first()  # Ignore PyFlakesBear
-        reached = self.browser.execute_script('return gaObject.results;')  # Ignore PyFlakesBear
+        order = Order.objects.order_by('-created').first()
+        order_positions = order.positions.all()
+        reached = self.browser.execute_script('return gaObject.results;')
+        require, transaction, *positions, send = reached
+
+        self.assertEqual(require, ['require', 'ecommerce'])
+        self.assertEqual(
+            transaction,
+            ['ecommerce:addTransaction', {
+                'id': order.fake_order_number, 'revenue': order.revenue,
+            }],
+        )
+        for pos, order_pos in zip(positions, order_positions):
+            self.assertEqual(
+                pos,
+                ['ecommerce:addItem', {
+                    'name': order_pos.name,
+                    'price': order_pos.price,
+                    'quantity': order_pos.quantity,
+                }],
+            )
+        self.assertEqual(send, ['ecommerce:send', None])
+
+    # @todo #771:60m Test Yandex eCommerce
+
+
+@tag('slow')
+@helpers.disable_celery
+@override_settings(DEBUG=True, INTERNAL_IPS=tuple())
+class YandexMetrika(helpers.SeleniumTestCase):
+
+    CART_LOCATOR = (By.CLASS_NAME, 'js-go-to-cart')
+
+    def setUp(self):
+        product_vendor_code = Product.objects.first().vendor_code
+        self.product_page = reverse('product', args=(product_vendor_code,))
+        self.category_page = reverse(
```

Comment:
`category_path` or `category_url` might be more good name. `*_path` one is the best, because here we mean part or an url without domain